README.md

```diff
@@ -72,41 +77,34 @@ To build for the Xen target, do:
 mirage configure --xen
 ```
 
-Once configuration is complete, you can install the OPAM packages required by
-this unikernel by:
-
-```
-make depend
-```
-
-## Building Mirage Applications
+### Building Mirage Applications
 
 The command:
 
 ```
-mirage build
+make
 ```
```

Comment:
here there should be some sentence... `The command make will compile your unikernel` or sth?